source/sphinx_extensions/plugin_directory/templates/plugin.rst

```diff
@@ -8,9 +8,9 @@
        <tr>
          <th scope="row">Description</th>
          <td>
-         	{% for line in plugin.description.splitlines() %}
-         	{{ line|urlize }}<br/>
-         	{% endfor %}
+           {% for line in plugin.description.splitlines() %}
```

Comment:
There were some tabs in this file, replaced with spaces.

Reply:
📑